metaci/repository/views.py

```diff
@@ -264,7 +264,11 @@ def get_or_create_release(event, tag, repo):
     release, _ = Release.objects.get_or_create(
         repo=repo,
         git_tag=tag,
-        defaults={"created_from_commit": event["head_commit"]["id"], "status": "draft"},
+        defaults={
+            "created_from_commit": event["head_commit"]["id"],
+            "status": "draft",
+            "change_case_template_id": 1,
+        },
```

Comment:
We aren't actually using this mechanism for creating a release based on a webhook from github, and I think it probably makes more sense to get rid of it rather than create a release with a default change case template which may or may not make sense.

Reply:
Removing this `"change_case_template_id": 1` causes metaci/repository/tests/test_views.py:385 to error out.

Reply:
I'm saying I think we should change this function so that it only finds existing releases but never creates a new one. We don't really want Release records to get created because of a github webhook, now that we are always creating them before a release tag gets created.

Reply:
I understand what you're saying but not sure what this would look like @davisagli